internal/config/cors.go

```diff
@@ -10,12 +10,22 @@ var _ defaulter = (*CorsConfig)(nil)
 type CorsConfig struct {
 	Enabled        bool     `json:"enabled" mapstructure:"enabled" yaml:"enabled"`
 	AllowedOrigins []string `json:"allowedOrigins,omitempty" mapstructure:"allowed_origins" yaml:"allowed_origins,omitempty"`
+	AllowedHeaders []string `json:"allowed_headers,omitempty" mapstructure:"allowed_headers" yaml:"allowed_headers,omitempty"`
```

Comment:
Maybe `json:allowedHeaders` to make it consistent with `allowedOrigins`?

```suggestion
	AllowedHeaders []string `json:"allowedHeaders,omitempty" mapstructure:"allowed_headers" yaml:"allowed_headers,omitempty"`
```

Reply:
Ah nice catch!

Reply:
(I don't actually know what this is used for, just spotted this because it's not consistent)